docs/src/main/tut/typeclasses/monoid.md

```diff
@@ -7,70 +7,138 @@ scaladoc: "#cats.kernel.Monoid"
 ---
 # Monoid
 
-`Monoid` extends the [`Semigroup`](semigroup.html) type class, adding an 
-`empty` method to semigroup's `combine`. The `empty` method must return a 
-value that when combined with any other instance of that type returns the 
-other instance, i.e.
+`Monoid` extends the power of `Semigroup` by providing an additional `empty` value.
 
-```scala
-(combine(x, empty) == combine(empty, x) == x)
-```
-    
-For example, if we have a `Monoid[String]` with `combine` defined as string 
-concatenation, then `empty = ""`.
+```tut:book:silent
+trait Semigroup[A] {
+  def combine(x: A, y: A): A
+}
 
-Having an `empty` defined allows us to combine all the elements of some 
-potentially empty collection of `T` for which a `Monoid[T]` is defined and 
-return a `T`, rather than an `Option[T]` as we have a sensible default to 
-fall back to.
+trait Monoid[A] extends Semigroup[A] {
+  def empty: A
+}
+```
 
-First some imports.
+This `empty` value should be an identity for the `combine` operation, which means the following equalities hold
+for any choice of `x`.
 
-```tut:silent
-import cats._
-import cats.implicits._
+```
+combine(x, empty) = combine(empty, x) = x
 ```
 
-Examples.
+Many types that form a `Semigroup` also form a `Monoid`, such as `Int`s (with `0`) and `Strings` (with `""`).
```

Comment:
Probably tangential to this PR, but I've read discussions on the variations of

`T`:
* has a
* is a
* forms a

`Typeclass[T]`, maybe this is something nice to have a sentence about? I don't care which phrase is used though (maybe a note that they're saying the same thing), and I'm guessing somewhere on http://typelevel.org/cats/typeclasses.html would be appropriate.